backend/services/product/internal/product/product.go

```diff
@@ -0,0 +1,116 @@
+package product
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/manaaan/ekolivs-oms/pkg/env"
+	"github.com/manaaan/ekolivs-oms/pkg/zettle"
+	"github.com/manaaan/ekolivs-oms/product/api"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type Service struct {
+	zettleService *zettle.Service
+	storeService  *Store
+}
+
+func New() (*Service, error) {
+	zettleService, err := zettle.New(zettle.ServiceNewParams{
+		ClientId: env.Required("ZETTLE_ORG_UUID"),
```

Comment:
This should probably be called ZETTLE_CLIENT_ID again, because it's not the ORG_UUID, which is a different, but similarly looking ID :)